<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Excel Save</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="1">
       <widget class="QListView" name="listCellsSave"/>
      </item>
      <item row="1" column="1">
       <widget class="QListView" name="listCellsAvailable"/>
      </item>
      <item row="1" column="3">
       <widget class="QListView" name="listTimesAvailable"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Compounds</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Cells</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Assays</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QListView" name="listCompoundsAvailable"/>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Conditions</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QListView" name="listConditionsSave"/>
      </item>
      <item row="3" column="5">
       <widget class="QListView" name="listAssaysSave"/>
      </item>
      <item row="1" column="5">
       <widget class="QListView" name="listAssaysAvailable"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Available</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QListView" name="listCompoundsSave"/>
      </item>
      <item row="1" column="4">
       <widget class="QListView" name="listConditionsAvailable"/>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Exposure Times</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QListView" name="listTimesSave"/>
      </item>
      <item row="2" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="bttnAddCompounds">
          <property name="text">
           <string>▼</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bttnDeleteCompounds">
          <property name="text">
           <string>▲</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="5">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="bttnAddAssays">
          <property name="text">
           <string>▼</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bttnDeleteAssays">
          <property name="text">
           <string>▲</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="4">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="bttnAddConditions">
          <property name="text">
           <string>▼</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bttnDeleteConditions">
          <property name="text">
           <string>▲</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="bttnAddCells">
          <property name="text">
           <string>▼</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bttnDeleteCells">
          <property name="text">
           <string>▲</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="bttnSave">
        <property name="text">
         <string>Save to Excel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>To Save</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="bttnAddTimes">
          <property name="text">
           <string>▼</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bttnDeleteTimes">
          <property name="text">
           <string>▲</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
